src/Sarif/Autogenerated/Fix.cs

```diff
@@ -32,7 +32,7 @@ public SarifNodeKind SarifNodeKind
         }
 
         /// <summary>
-        /// A message that describes the proposed fix, enabling viewers to present the proposed change to an end user.
+        /// A plain text message that describes the proposed fix, enabling viewers to present the proposed change to an end user.
```

Comment:
>plain text [](start = 14, length = 10)

The OASIS schema is wrong and the SDK's version is right. This property is a `message` object, not a plain text message.